src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -326,7 +328,7 @@ static function ( $event ) use ( $start, $end, $use_site_timezone, $site_timezon
 		 * Multi-day events will always appear on the second day and forward, back-fill if they did not make the
 		 * cut (of events per day) on previous days.
 		 */
-		$this->backfill_multiday_event_ids( $this->grid_events );
+		 $this->backfill_multiday_event_ids( $this->grid_events );
```

Comment:
* :warning: Found precision alignment of 1 spaces.
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 70e657badc2bb20a23ca6d70e565c610</sub>